app/src/main/java/org/sopt/sample/data/remote/ResponseFollowerListDto.kt

```diff
@@ -0,0 +1,29 @@
+package org.sopt.sample.data.remote
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ResponseFollowerListDto(
+    val page: Int,
+    val per_page: Int,
```

Comment:
```suggestion
    @SerialName("per_page")
    val perPage: Int,
```
저는 이렇게 코틀린 컨벤션과 json 속성 이름이 다를 경우에 `@SerialName` 을 사용하곤 합니다..!